pages/production.html

```diff
@@ -32,6 +32,10 @@
 This profile will compile, test and package your Java application with all productions settings. It will also trigger a "gulp build", so your static resources are optimized.
 </p>
 <p>
+Earlier versions of JHipster put these optimized assets in <code>src/main/webapp/dist</code>, as a child directory of the "dev" assets. As of 3.0, these have been separated and
```

Comment:
I will re-phrase this - we don't talk about "earlier versions" or we will end up with tons of older stuff